example/package.json

```diff
@@ -13,7 +14,7 @@
     "@babel/core": "^7.0.0",
     "mkdirp": "^0.5.1",
     "react": "16.8.6",
-    "react-native": "0.60.6",
+    "react-native": "0.61.5",
```

Comment:
Avoid bumping to 61. Two possible solutions: 
1) In `test-app-native-modules.gradle`, use two different commands to find the path to CLI binaries (see iOS's autolinking implementation)
2) Bump CLI tools independently.